server/src/main/java/io/crate/breaker/ConcurrentRamAccounting.java

```diff
@@ -39,23 +42,46 @@ public final class ConcurrentRamAccounting implements RamAccounting {
     private final AtomicLong usedBytes = new AtomicLong(0L);
     private final LongConsumer reserveBytes;
     private final LongConsumer releaseBytes;
+    private final String label;
+    private final int operationMemoryLimit;
 
-    public static ConcurrentRamAccounting forCircuitBreaker(String label, CircuitBreaker circuitBreaker) {
+    public static ConcurrentRamAccounting forCircuitBreaker(String label, CircuitBreaker circuitBreaker, int operationMemoryLimit) {
```

Comment:
Does it make sense for tests to  have another method without the limit to avoid passing 0 in some places?

Reply:
I'd rather make it a bit painful and force callers to provide a value. Ideally we want the session setting to be used.